.travis.yml

```diff
@@ -24,9 +24,11 @@ install:
 - conda install -c conda-forge protobuf=3.1.0
 - conda install -c omnia mdtraj
 - pip install tensorflow==0.12.1
+- pip install yapf
```

Comment:
it's a good idea to freeze yapf - different versions sometimes give different results (we've seen this internally).